framework/logging.cpp

```diff
@@ -1834,7 +1834,7 @@ void YamlLogger::maybe_print_slice_resource_usage(int fd, int slice)
             indent_spaces().data(),
             format_duration(utime, FormatDurationOptions::WithoutUnit).c_str(),
             format_duration(stime, FormatDurationOptions::WithoutUnit).c_str(),
-            (utime + stime) * 100.0 / runtime,
+            std::abs((utime + stime) * 100.0 / runtime),
```

Comment:
No, this doesn't look right. Why are the numbers negative in the first place?

`utime` and `stime` come from the `usage`, so they shouldn't be negative. So my guess it's `runtime`:
```c++
    auto runtime = slices[slice].endtime - sApp->current_test_starttime;
```
That's probably because we didn't set `endtime`. Something I thought I'd fixed... 606566be57deac84673f0cb307e65ba9a9cc7ca9, ah, interrupted tests only. I'd rather fix similarly there.